WT3 Fixed Bed Unit

adam-a-a opened the pull request:
## Fixes/Addresses:

## Summary/Motivation:
Adding fixed_bed_zo unit with gravity_basin and pressure_vessel subtypes.

## Changes proposed in this PR:
- add fixed_bed_zo unit with tests
- merge fixed_bed_gravity_basin and fixed_bed_pressure_vessel yaml files into one fixed_bed yaml

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

adam-a-a committed:
adding fixed_bed_zo with gravity_basin and pressure_vessel subtypes

andrewlee94 reviewed (dismissed):
Looks good - I do wonder about the two sub-types as they both have the same values, but that is probably lack of data.

andrewlee94 commented:
@adam-a-a It looks like you have a couple of conflicts to fix.

adam-a-a commented:
> Looks good - I do wonder about the two sub-types as they both have the same values, but that is probably lack of data.

Yes, I added those because no removal data were included. So I added removal rates for BOD with fairly arbitrary values. We can go back and refine when we know more.

adam-a-a committed:
resolve conflicts after Merge branch 'main' into wt3_fixed_bed

adam-a-a committed:
Merge branch 'main' into wt3_fixed_bed

adam-a-a committed:
Merge branch 'main' into wt3_fixed_bed
